corehq/apps/app_manager/models.py

```diff
@@ -2173,15 +2177,16 @@ class CaseSearchProperty(DocumentSchema):
     Case properties available to search on.
     """
     name = StringProperty()
-    label = DictProperty()
+    label = LabelProperty()
     appearance = StringProperty(exclude_if_none=True)
     input_ = StringProperty(exclude_if_none=True)
     default_value = StringProperty(exclude_if_none=True)
-    hint = DictProperty()
+    hint = LabelProperty()
     hidden = BooleanProperty(default=False)
     allow_blank_value = BooleanProperty(default=False)
     exclude = BooleanProperty(default=False)
     required = StringProperty(exclude_if_none=True)
+    required_message = LabelProperty()
     validation = SchemaListProperty(CaseSearchValidationCondition)
```

Comment:
Should this be `validations`? I didn't notice in the last PR, but seeing `for validation in prompt.validation` in this one tripped a switch in my brain.